core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/drill/DrillDatabaseConnectionService.java

```diff
@@ -27,4 +27,9 @@ public String getDriver(DrillDatabaseConnection databaseConnection) {
     public Class<DrillDatabaseConnection> appliesTo() {
         return DrillDatabaseConnection.class;
     }
+
+    @Override
+    public String refactorLimitAndOffset(DrillDatabaseConnection databaseConnection, String query) {
+        return query;
+    }
```

Comment:
All XDatabaseConnectionService's extend either DatabaseConnectionService or SchemaDatabaseConnectionService (which extends DatabaseConnectionService). You could implement the default behavior in the DatabaseConnectionService class